packages/core/strapi/lib/commands/utils/commander.js

```diff
@@ -67,10 +68,21 @@ const promptEncryptionKey = async (thisCommand) => {
 };
 
 /**
- * hook: require a confirmation message to be accepted
+ * hook: require a confirmation message to be accepted (unless --force option is used)
+ *
+ * @param {string} message The message to confirm with user
+ * @param {object} options Additional options
  */
 const confirmMessage = (message) => {
-  return async () => {
+  return async (command) => {
+    // if we have a force option, assume yes
+    const opts = command.opts();
+    if (opts?.force) {
```

Comment:
Do we want to ensure it's true or does a truth value is enough?

Reply:
But I'll make --force a re-usable option as well, so nobody could accidentally implement it differently.